### DISCUSSION TITLE
Displaying Persian/Jalali (moment-jalaali) dates in react apexchart

### DISCUSSION THREAD

**Discussion by AvaaStack** [apexcharts/apexcharts.js#2735] [Q&A]
Could you guide me on how to make a "**Datetime X-Axis**" chart with **react Apexchart.js** when my dates are in **Persian**? at last, I should make a "brushChart" with Datetime X-Axis, if you have a similar project that can help me please send link of them

I had these codes below and I tried "defaultLocale" and "name" with both "Per" and "Fa" but none of them worked and getting a error and the app crashed.

`chart: {
  id: 'reactchart-example',
  defaultLocale: 'Fa',
  locales: [
    {
      name: 'per',
      options: {
        months: [
          'دی',
          'بهمن',
          'اسفند',
          'فروردین',
          'اردیبهشت',
          'خرداد',
          'تیر',
          'مرداد',
          'شهریور',
          'مهر',
          'آبان',
          'آذر',
        ],
        shortMonths: ['دی', 'بهم', 'اسف', 'فرو', 'ارد', 'خرد', 'تیر', 'مرد', 'شهر', 'مهر', 'آبا', 'آذر'],
        days: ['یکشنبه', 'دوشنبه', 'سه شنبه', 'چهارشنبه', 'پنج شنبه', 'جمعه', 'شنبه'],
        shortDays: ['یک', 'دو', 'سه', 'چهار', 'پنج', 'جمع', 'شنب'],
        toolbar: {
          download: 'دانلود SVG',
          selection: 'انتخاب',
          selectionZoom: 'انتخاب بزرگنمایی',
          zoomIn: 'بزرگنمایی',
          zoomOut: 'کوچک نمایی',
          pan: 'جابجایی',
          reset: 'بازگرداندن',
        },
      },
    },
  ],
},`

**Comment by matinwd** [comment]
Did you solve this problem?

**Comment by AvaaStack** [comment]
Datetime X-Axis this type is just for Christian calendar if you need persian calendar  use text X-Axis and send your dates in persian from back

**Comment by matinwd** [comment]
You just need to use formatter: (){}
It does the magic
